lib/mocha/mock.rb

```diff
@@ -327,7 +327,7 @@ def method_missing(symbol, *arguments, &block)
         else
           message = @unexpected_invocation.short_description
         end
-        raise ExpectationErrorFactory.build(message, caller)
+        raise ExpectationErrorFactory.build("unexpected invocation: #{message}", caller)
```

Comment:
It would be helpful to explain in the commit note that you're planning to remove the `UnexpectedInvocation` class in a later commit, otherwise it seems a bit odd to move the "unexpected invocation: " string out of that class.

Reply:
I've resolved this in my local version of the branch.